docs/sources/alerting/notifications.md

```diff
@@ -75,20 +75,22 @@ able to access the image.
 
 {{< imgbox max-width="40%" img="/img/docs/v4/slack_notification.png" caption="Alerting Slack Notification" >}}
 
-To set up slack you need to configure an incoming webhook url at slack. You can follow their guide on how
+To set up Slack, you need to configure an incoming Slack webhook URL. You can follow their guide on how
```

Comment:
```suggestion
To set up Slack, you need to configure an incoming Slack webhook URL. You can follow the Slack guide on how
```

Reply:
To be honest, I think the older wording is better, since the new wording sounds overly repetitive to my ears (3x Slack in the same sentence). Besides, wording wasn't introduced by my PR, so if you want to change it around I think it's better to make a new PR.